mycore-webtools/src/main/vue/texteditor/src/views/TextEditorView.vue

```diff
@@ -0,0 +1,134 @@
+<script setup lang="ts">
+import {type Ref, ref, watch} from "vue";
+import {ContentHandlerSelector} from "@/apis/ContentHandlerSelector";
+import TextEditor from "@/components/TextEditor.vue";
+
+const props = defineProps<{
+  type: string,
+  id: string
+}>();
+let model: Ref<Content> = ref({
+  data: "",
+  type: ""
+});
+let loading: any = ref(true);
+let success: any = ref();
+let error: any = ref();
+let updateEnabled = ref(false);
+let writeAccess = ref(false);
+let undoKeyPressed = false;
+let originalModel: string = "";
+
+watch(() => model.value.data, async (newModel) => {
+  // prevent ctrl+z clearing the textarea
+  setTimeout(() => {
```

Comment:
use nextTick() or even better prevent the ctrl+z browser function by catching the event and return false